Dockerfile

```diff
@@ -19,6 +19,18 @@ RUN DEBIAN_FRONTEND=noninteractive apt update && apt install -y \
 
 RUN curl --proto '=https' --tlsv1.2 -L https://github.com/mikefarah/yq/releases/download/v4.33.3/yq_linux_amd64 | install -m 755 /dev/stdin /bin/yq && yq --version | grep yq
 
+# Install node
+RUN curl --fail -sSf https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
```

Comment:
Do the front-end builds and back-end builds depend on each other in any way? Maybe it makes sense to _not_ share a docker file if they are separate, leading to faster builds and smaller set of dependencies.

Reply:
I think that should be possible. I just wanted to get something going fast, so I decided to go this route. I think, separating the two shouldn't be too difficult. But maybe I should just give this a try now, since this would resolve all your comments below, about the different stages etc.

Reply:
Yes, thanks!